serialization/Elements/sdoc.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
for an SPDX document, the `name` is mandatory and is missing here

Reply:
Good catch, thanks.